### PR TITLE
Add `StringPrefix` type

### PR DESCRIPTION
**Pull request by pvdrz**

### CODE REVIEW

File: packable/packable-derive-test/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "packable-derive-test"
-version = "0.3.3"
+version = "0.3.4"
```
Comment (thibault-martinez) [inline packable/packable-derive-test/Cargo.toml:3]: Sounds more like a minor to me tbh

File: packable/packable/Cargo.toml [hunk 2]
```diff
@@ -19,7 +19,7 @@ usize = [ ]
 autocfg = { version = "1.0.1", default-features = false }
 
 [dependencies]
-packable-derive = { version = "=0.3.3", path = "../packable-derive", default-features = false }
+packable-derive = { version = "=0.3.4", path = "../packable-derive", default-features = false }
 
 primitive-types = { version = "0.10.1", default-features = false, optional = true }
```
Comment (thibault-martinez) [inline packable/packable/Cargo.toml:24]: Can you bump this to the max please ? I'm trying to bump it in other crates but it's incompatible with packable's one

File: packable/packable/tests/string_prefix.rs [hunk 3]
```diff
@@ -0,0 +1,135 @@
+// Copyright 2021-2022 IOTA Stiftung
```
Comment (thibault-martinez) [inline packable/packable/tests/string_prefix.rs:1]: Just 2022

File: packable/packable-derive-test/Cargo.toml (hunk 1 above)
Comment (pvdrz) [reply packable/packable-derive-test/Cargo.toml:3]: indeed: https://doc.rust-lang.org/cargo/reference/semver.html#item-new

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**